lib/_http_incoming.js

```diff
@@ -400,6 +399,16 @@ function _addHeaderLine(field, value, dest) {
     } else {
       dest['set-cookie'] = [value];
     }
+  } else if (this.joinDuplicateHeaders) {
+    // RFC 9110 https://www.rfc-editor.org/rfc/rfc9110#section-5.2
+    // https://github.com/nodejs/node/issues/45699
+    // allow authorization multiple fields
+    // Make a delimited list
+    if (typeof dest[field] === 'string') {
+      dest[field] += ', ' + value;
```

Comment:
The docs and the implementation differs. Is this the correct one, or space comes before comma?

Reply:
This is correct the doc has a typo